### PR TITLE
Ang init

### PR DESCRIPTION
**Pull request by MaxDrozdov**
Loading data from json

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,56 +2,24 @@
...
-    <title>Testang1 by MaxDrozdov</title>
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="stylesheet" type="text/css" href="stylesheets/normalize.css" media="screen">
-    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
-    <link rel="stylesheet" type="text/css" href="stylesheets/github-light.css" media="screen">
+    <title>Testang1</title>
+    <script src="libs/angular/angular.min.js"></script>
+    <script src="libs/angular/angular-resource.min.js"></script>
+    <script src="js/app.js"></script>
```
Comment (NE05KY) [inline index.html]: inject your code only after all libraries

Comment (MaxDrozdov) [inline index.html]: Done

File: index.html [hunk 2]
```diff
@@ -2,56 +2,31 @@
...
-      </footer>
-
-    </section>
-
-  
+  </head>
+  <body ng-app="baseApp">
+    <div ng-controller="newsCtrl as news">
+      <div ng-repeat="new in newsData">
+        <pre style="margin: 15px">
```
Comment (NE05KY) [inline index.html]: Don't inline styles, move it to styles.css

Comment (MaxDrozdov) [inline index.html]: Done

**Pull request merged**